### ISSUE TITLE
Is this repo still mantained?

### ISSUE THREAD

**Issue by dfit99** [magro/kryo-serializers#136]
Is this repo still mantained?

**Comment by magro** [comment]
Not really, and I'm sorry for that!

I'd be happy if someone would be willing to take over maintainership.

**Comment by magro** [comment]
Would you be open for that, @theigl? Not sure about the current state of Kryo and how kryo-serializers still fit.

**Comment by theigl** [comment]
Hi @magro! kryo-serializers is still relevant for Kryo 5 because there are no built in serializers for unmodifiable and synchronized collections. Also Guava and JodaTime are still widely used. I think I can take over maintaining this library. I'm currently on vacation and will get back to you in 2 weeks.

**Comment by magro** [comment]
Sounds great, @theigl! I've invited you as collaborator now. Enjoy your vacation! :v:

**Comment by theigl** [comment]
@magro: Thanks a lot! 

What permissions do I need to perform a release? I guess you will need to add me somewhere? Is the release process the same as for [Kryo](https://github.com/EsotericSoftware/kryo/wiki/HowTo-Release)?

**Comment by magro** [comment]
Exactly, you need to get push permissions for sonatype for the related groupId. What's you sonatype username, @theigl?

**Comment by magro** [comment]
I just created https://issues.sonatype.org/browse/OSSRH-93213

**Comment by magro** [comment]
@theigl As https://issues.sonatype.org/browse/OSSRH-93213 is resolved you could try to publish something...

**Comment by theigl** [comment]
Thanks @magro!
